edgar-doctor/src/components/app/dashboardPages/appointments/AppointmentsCards.tsx

```diff
@@ -15,8 +15,12 @@ import { type AppointmentType } from 'types/app/dashboard/appointments/appointme
 const AppointmentsCards = ({ appointments }: { appointments: AppointmentType[] }): JSX.Element => {
 	const [pageIndexNext, setPageIndexNext] = useState(1);
 	const [pageIndexOld, setPageIndexOld] = useState(1);
-	const upcomingAppointments = appointments.filter((appointment) => appointment.endDate > new Date().getTime());
-	const pastAppointments = appointments.filter((appointment) => appointment.endDate <= new Date().getTime());
+	const upcomingAppointments = appointments.filter(
+		(appointment) => appointment.endDate > new Date().getTime() && appointment.appointmentStatus !== 'CANCELED',
```

Comment:
Not enough because the canceled appointments can be either "CANCELED" or "CANCELED_DUE_TO_REVIEW"